src/views/ContactsMenuAvailability.vue

```diff
@@ -50,7 +50,10 @@ export default {
 		},
 	},
 	data() {
-		const initialAttendee = AttendeeProperty.fromNameAndEMail(this.userId, this.userEmail)
+		const initialAttendee = AttendeeProperty.fromNameAndEMail(
+			this.userDisplayName,
```

Comment:
No fallback to userid? We're safe?

Reply:
Yeah, it will always be set for users and the action is only shown for users (but not for regular contacts). 

Background: The contacts menu API response contains a display name which is synced from the corresponding contact's `FN`. For users, the `FN` property is synced from their display name. Now, the display name's scope cannot be made private so it will always be present. It will at least be set to local so a display name will always be synced to the SAB.